src/io/flutter/utils/ElementIO.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
nit: copyrights can be 2019 now

Reply:
Update all files or just newly added ones?

Reply:
From discussion, it looks like we leave copyrights on files as the year the file was created, so new files will be copyright 2019, files I created in 2018 will be copyright 2018, and other files should have their copyrights untouched.